backend/tests/unit/records/marshmallow/literature/test_ui.py

```diff
@@ -279,3 +279,31 @@ def test_documents_doesnt_contain_error_from_fulltext(
     serializer = LiteratureDetailSchema()
     serialized = serializer.dump(entry_data).data
     assert "_error" not in serialized
+
+def test_filter_non_pdg_keywords():
+    data = {
+        "keywords": [
+            { "schema": "PDG", "source": "PDG", "value": "M013WX" },
+            { "source": "publisher", "value": "Strong Interactions" },
+        ]
+    }
+    expected_data = [
+        { "source": "publisher", "value": "Strong Interactions" }
+    ]
+    serializer = LiteratureDetailSchema()
+    serialized = serializer.dump(data).data
+    assert serialized["keywords"] == expected_data
+
+def test_filter_pdg_keywords():
+    data = {
+        "keywords": [
+            { "schema": "PDG", "source": "PDG", "value": "M013WX" },
+            { "source": "publisher", "value": "Strong Interactions" },
+        ]
+    }
+    expected_data = [
+        { "pid": "M013WX", "value": "f(2)'(1525) WIDTH" }
```

Comment:
hm is that correct? it is not test case `data`

Reply:
It is the correct result of  this function to filter out and map pdg keyword id to its' description `pdg_keywords.append({'pid': pdg_identifier, 'value': PDG_IDS_TO_DESCRIPTION_MAPPING[pdg_identifier]})`, and the tests' `serialized["pdg_keywords"]` equals exactly this, so I think it's correct, but I might be missing something.

Reply:
ok, I overlooked it but I think it's wrong to call the keyword value `pid` cause IMO that should be reserved for persistent identifiers of our records. I would add instead an entry with keys (`value`, `description`) instead